src/ray/raylet/actor.h

```diff
@@ -0,0 +1,26 @@
+#ifndef ACTOR_H
+#define ACTOR_H
+
+#include "ray/id.h"
+
+namespace ray {
+class ActorInformation {
+public:
+  /// \brief ActorInformation constructor.
```

Comment:
This is the reason why I think doxygen is not that useful.  It encourages excessive, unnecessary documentation.